gilbreth_grasp_planning/scripts/test_pose_execution.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
```

Comment:
I thought this whole script was already a part of the repo, it's showing as a new script.

Reply:
This is a slightly different file I used to test the goal tolerance. I will delete it.